assignment-1/pkg/controller/common.go

```diff
@@ -0,0 +1,17 @@
+package controller
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+func GetSnapshotClassname() string {
+	return "csi-hostpath-snapclass"
+}
```

Comment:
Instead of hardcoding snapshotclass, we can find it by listing all the vsnapshotclasses and comparing the provisioned. The one with the same provisioned as PVC should be used

Reply:
fixed